src/Models/user/Transformers/UserTransformer.ts

```diff
@@ -113,7 +115,7 @@ export default class UserTransformer extends BaseTransformer {
         return this.collection(user, 'userSubscriptions', new UserSubscriptionTransformer());
     }
 
-    includeUserSubscription(user: UserInterface): UserSubscriptionInterface | null {
-        return this.item(user, 'userSubscriptions', new UserSubscriptionTransformer());
+    includeUserSubscription(user: UserInterface): Array<UserSubscriptionInterface> | null {
+        return this.collection(user, 'userSubscriptions', new UserSubscriptionTransformer());
```

Comment:
Was there any specific reason, why did you decide not to use `includeUserSubscriptions`, which after your change is returning exactly the same thing as `includeUserSubscription`?

Reply:
No just a missed change to make